ipatests/test_integration/test_epn.py

```diff
@@ -243,6 +249,20 @@ def test_EPN_config_file(self):
         ck = "4c207b5c9c760c36db0d3b2b93da50ea49edcc4002d6d1e7383601f0ec30b957"
         assert cmd2.stdout_text.find(ck) == 0
 
+    def test_EPN_connection_refused(self):
+        """Test EPN behavior when the configured SMTP is down
+        """
+
+        self.master.run_command(["systemctl", "stop", "postfix"])
+        (stdout_text, unused, rc) = self._check_epn_output(
+            self.master, mailtest=True,
+            raiseonerr=False, validatejson=False
+        )
+        self.master.run_command(["systemctl", "start", "postfix"])
+        assert "IPA-EPN: Could not connect to the configured SMTP server." in \
```

Comment:
either drop the trailing period in the string or make it a colon.

Reply:
also check for error string in stderr, not stdout.